pages/documentation/showcase.md

```diff
@@ -93,7 +91,6 @@ If your project is currently using USWDS and you would like to see it included i
 - [Manufacturing USA](https://www.manufacturing.gov/)
 - [Medicaid](https://www.medicaid.gov/)
 - [Method Cards - 18F](https://methods.18f.gov/)
-- [Move.mil — Official DOD Moving Portal](https://www.move.mil/)
 - [MyMedicare](https://www.mymedicare.gov/)
 - [NASA General Coordinates Network (GCN)](https://gcn.nasa.gov)
 - [NASA Glenn Research Center](https://www1.grc.nasa.gov)
```

Comment:
@finekatie - while we're updating the page can we also correct the spelling of "Satellite" in "[National Environmental Sattelite Data and Information Service (NOAA)](https://www.nesdis.noaa.gov/)" (This issue was also flagged in the Siteimprove report.)

Reply:
Hey there, @sarah-sch , I can, but since this isn't a broken link, do I need to update the Issue, etc.?

Reply:
Discussed with @finekatie on 4/2 and updated issue

Reply:
Confirming this is fixed in 64f6c0c0.